pkgs/development/python-modules/molecule/plugins.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonRelaxDepsHook
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "molecule-plugins";
+  version = "23.4.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-NMR+4sEcNbowyoTqaEwe4Wac9+WNIZesnb/L9C0KG3s=";
+  };
+
+  # reverse the dependency
+  pythonRemoveDeps = [
+    "molecule"
+  ];
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+    setuptools-scm
+  ];
+
```

Comment:
Can we set `passthru.optional-dependencies`?
https://github.com/ansible-community/molecule-plugins/blob/7eab71d0da547914e46a1110c56d4ac949b9bce9/pyproject.toml#L51

Reply:
https://github.com/NixOS/nixpkgs/pull/241313/commits/1d37534b1c1ac6259ee189a6c3971763765a1fc4